lib/cryptoexchange/models/order.rb

```diff
@@ -0,0 +1,17 @@
+module Cryptoexchange
+  module Models
+    class Order
+      attr_accessor :price, :amount, :timestamp
+
+      def initialize(args={})
+        @price      = args[:price]
+        @amount     = args[:amount]
+        @timestamp  = args[:timestamp] || nil
+      end
+
+      def format_as_list
```

Comment:
@tmlee Actually, why do we need to do this? Can't we just return an array of `Cryptoexchange::Models::Order` ?

Reply:
@dominicwong617 That is possible. We just need to decide what end result we want to provide. Though i think returning as `Order` would give the user best flexibility how they want to store the results.

Reply:
@dominicwong617 now giving the user the choice to call this if they want to

Reply:
@tmlee let's just return an array of object as asks and bids, this choice doesn't add much benefits